app/Jobs/MakeUnattachedGovernmentsInActive.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Jobs;
+
+use App\Models\Government;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldBeUnique;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Queue\SerializesModels;
+
+class MakeUnattachedGovernmentsInActive implements ShouldQueue
+{
+    use Dispatchable, InteractsWithQueue, Queueable, SerializesModels;
+
+    /**
+     * Create a new job instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        //
+    }
+
+    /**
+     * Execute the job.
+     *
+     * @return void
+     */
+    public function handle()
+    {
+        $governments = Government::get();
```

Comment:
1- Move logic to controller
2- Check other possible solutions to do a check and a bulk update instead of retrieving the entire table, which is not efficient at all.